### PR TITLE
[DOCFIX] Add some key metrics to the tuning docs

### PR DESCRIPTION
**Pull request by yuzhu**
### What changes are proposed in this pull request?

Add descriptions of key metrics relevant to tuning

### Why are the changes needed?
Make it more clear and easy to find critical metrics

### Does this PR introduce any user facing changes?
Doc  changes

### CODE REVIEW

File: docs/en/operation/Performance-Tuning.md [hunk 1]
```diff
@@ -363,3 +373,79 @@ persistence again when renamed.
...
+
+## Frequently Seen Performance Issues
+
+This section lists a set of common performance issues and possible reasons and diagnostics steps.
+It is a good place to start when you have a performance issue and may lead to an answer quickly if your symptom matches one of those described here. 
+
+### Slow Queries / Overall performance 
+Unexpectedly large `Cluster.BytesReadUfs` metric is observed.
+
+When we are going to UFS for the data, we are sacrificing performance and incurring additional cost. This is usually the biggest red flag.
```
Comment (apc999) [inline docs/en/operation/Performance-Tuning.md:385]: let's avoid using "we" but "Alluxio" to be more precise?

File: docs/en/operation/Performance-Tuning.md [hunk 2]
```diff
@@ -363,3 +373,79 @@ persistence again when renamed.
...
+1. Worker rpc thread max
+   * alluxio.worker.network.block.reader.threads.max
+   * alluxio.worker.network.block.writer.threads.max
+These two settings control the concurrency levels of the reader and writer threads. 
+   * alluxio.worker.remote.io.slow.threshold  
+This setting controls when a remote io is considered slow. If a remote io is slower than this, check the worker log for messages
+1. Worker timeout
+   * Check client log for any worker timeout and check worker log for any dead worker
+
+### Slow metadata sync
```
Comment (apc999) [inline docs/en/operation/Performance-Tuning.md:425]: can you give a example here when we can tell we have slow metadata sync?

File: docs/en/operation/Scalability-Tuning.md [hunk 3]
```diff
@@ -45,6 +45,19 @@ backup duration.
 time proportional to the number of files in the subtree being operated on. This should not impact
 user experience unless the subtree is significantly large (> 10k files).
 
+Other metrics related to the number of files and the number of blocks
+* Master.TotalPath
+* Master.InodeHeapSize (Estimate only)
+* Master.TotalBlocks
```
Comment (apc999) [inline docs/en/operation/Scalability-Tuning.md:51]: I think this one is replaced by Master.UniqueBlocks and Master.BlockReplicaCount?

File: docs/en/operation/Performance-Tuning.md [hunk 4]
```diff
@@ -363,3 +373,79 @@ persistence again when renamed.
...
+1. Using `jps` to ensure job master and job worker processes are running
+1. `alluxio jobs ls` to see if there are active jobs
+1. Check the master log to see if the jobs are triggered
+1. Check if we have reached job service capacity, increase 
+`alluxio.job.master.job.capacity` if necessary
+1. Adjust `alluxio.job.worker.threadpool.size` to increase concurrency (this might affect worker performance)
+
+### OOM of Alluxio processes
+1. Alluxio process can get killed by system OOM killer and die silently
+ * Check dmesg -T | egrep -i 'killed process'
```
Comment (apc999) [inline docs/en/operation/Performance-Tuning.md:446]:
```suggestion
 * Check `dmesg -T | egrep -i 'killed process'`
```

**Review by apc999 — CHANGES_REQUESTED** [review]
comments left. Thanks for the improvement!

**Comment by yuzhu** [comment]
@apc999 PTAL again thanks.

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**